Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Home.kt

```diff
@@ -67,107 +64,139 @@ import androidx.compose.ui.unit.Dp
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.util.lerp
 import androidx.core.os.ConfigurationCompat
+import androidx.navigation.NavBackStackEntry
+import androidx.navigation.NavController
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.compose.KEY_ROUTE
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.currentBackStackEntryAsState
+import androidx.navigation.compose.getBackStackEntry
+import androidx.navigation.compose.navigate
+import androidx.navigation.compose.rememberNavController
 import com.example.jetsnack.R
-import com.example.jetsnack.ui.components.JetsnackScaffold
 import com.example.jetsnack.ui.components.JetsnackSurface
 import com.example.jetsnack.ui.home.cart.Cart
 import com.example.jetsnack.ui.home.search.Search
 import com.example.jetsnack.ui.theme.JetsnackTheme
-import com.google.accompanist.insets.ProvideWindowInsets
 import com.google.accompanist.insets.navigationBarsPadding
 
-@Composable
-fun Home(onSnackSelected: (Long) -> Unit) {
-    val (currentSection, setCurrentSection) = rememberSaveable {
-        mutableStateOf(HomeSections.Feed)
+fun NavGraphBuilder.home(
```

Comment:
I like these to be called `addNameGraph`, to make it clearer when calling the fun

Reply:
Fixed